airflow/cli/commands/dag_command.py

```diff
@@ -245,17 +251,11 @@ def set_is_paused(is_paused: bool, args) -> None:
         should_apply = ask_yesno(question)
 
     if should_apply:
-        dags_models = [DagModel.get_dagmodel(dag.dag_id) for dag in dags]
-        for dag_model in dags_models:
-            if dag_model is not None:
-                dag_model.set_is_paused(is_paused=is_paused)
+        for dag_model in matched_dags:
+            dag_model.set_is_paused(is_paused=is_paused)
 
         AirflowConsole().print_as(
-            data=[
-                {"dag_id": dag.dag_id, "is_paused": dag.get_is_paused()}
-                for dag in dags_models
-                if dag is not None
-            ],
+            data=[{"dag_id": dag.dag_id, "is_paused": not dag.get_is_paused()} for dag in matched_dags],
```

Comment:
```suggestion
            data=[{"dag_id": dag.dag_id, "is_paused": not dag.get_is_paused()} for dag in matched_dags],
```
Why the use of `not dag.get_is_paused`?

Reply:
@ephraimbuddy 
Because when we query matching dags in:
https://github.com/apache/airflow/blob/d296cad70c7dbffc8a4e5e073188eb72034af1de/airflow/cli/commands/dag_command.py#L238
We get DagModel objects from this query, and they have the attribute `is_paused=True`

And then we unpause these Dags at:
https://github.com/apache/airflow/blob/d296cad70c7dbffc8a4e5e073188eb72034af1de/airflow/cli/commands/dag_command.py#L255
The `set_is_paused` method of `DagModel` don't actually update the `is_paused` instance attribute, it only update the value in the metadata db. 

Hence if we do not put `not dag.get_is_paused()`, if one unpauses dags, the console will still print `is_paused=True` like this:
```
dag_id               | is_paused
=====================+==========
future_schedule_none | True     
```

Reply:
I guess I can remove the `not` here. But then if one unpauses dags
The console will print the `dag_id` and the `is_paused` state as `True`, which is a bit misleading since if a user executes unpause command, they should see that the dag has been unpaused on their screen?

Reply:
Got it